src/sentry/services/hybrid_cloud/app/impl.py

```diff
@@ -32,5 +143,18 @@ def find_installation_by_proxy_user(
 
         return self.serialize_sentry_app_installation(installation, sentry_app)
 
+    def trigger_sentry_app_action_creators(
+        self, *, fields: List[Mapping[str, Any]], install_uuid: str | None
+    ) -> AlertRuleActionResult:
+        try:
+            install = SentryAppInstallation.objects.get(uuid=install_uuid)
+        except SentryAppInstallation.DoesNotExist:
+            return dict(success=False, message="Installation does not exist")
+        result = alert_rule_actions.AlertRuleActionCreator.run(
```

Comment:
This looks like the action creator that is going to send a request out to the sentryapp external site. Would we be sending those out from the control silo now?

Reply:
Yes, this was @mikejihbe 's concern as well.  The answer is, yes.  That's what would happen.

My reply is basically -- the integrations and alerts systems need some pretty thoughtful work to be fully run in the region silo, *specifically because a lot of the configuration is in the control silo still*.    It's not trivial; these action creators are recursive, there are multiple other mediators involved, and they all want prolific access to the underlying ORMs.  

As it stands much of our integrations logic will also be routing through control silo for the proxy and handling shared configuration.

There is a chunk of work probably to think and flatten this for region silo RPC.  It may even involve replication of certain models and logic, so that more of it can be local.  The tension is -- it's risky to change currently any of this for various reasons.   Number one being that to even get a single silo test passing, a lot of changes need to exist, and those changes need to be safe.  

I think it would be more correct, to consider refactoring this, *after* a safe deploy with passing region silo tests.  If you want to hold me to doing a deeper refactoring here to make region alerts, I am willing to do that in a separate follow up PR, that can be smaller, and still be validated by existing stable region silo tests.  Placing it in this PR increases the delta and the risk, and we've already slipped up twice.